plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts

```diff
@@ -260,4 +287,51 @@ test.describe( `${ blockData.name } Block`, () => {
 		await expect( minusButton ).toBeHidden();
 		await expect( plusButton ).toBeHidden();
 	} );
+
+	test( 'has the stepper mode working on the frontend with min, max, and step attributes', async ( {
+		admin,
+		editor,
+		blockUtils,
+		page,
+	} ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( { name: 'woocommerce/single-product' } );
+
+		await blockUtils.configureSingleProductBlock();
+
+		await blockUtils.enableStepperMode();
+		await editor.publishAndVisitPost();
+
+		await blockUtils.setMinMaxAndStep( {
+			min: 2,
+			max: 10,
+			step: 2,
+		} );
+
+		const minusButton = page.locator(
+			'.wc-block-components-quantity-selector__button--minus'
+		);
+		const plusButton = page.locator(
+			'.wc-block-components-quantity-selector__button--plus'
+		);
```

Comment:
Thanks for your great suggestion! I implemented the aria-label only on the frontend since the buttons aren’t clickable on the editor side.

I addressed your feedback with commit 87c28878c0b3b2d9afd4a5d7200b634ff6664cbb. Let me know if you have any other comments or suggestions.